src/MySqlConnector/MySqlParameter.cs

```diff
@@ -12,6 +12,8 @@ namespace MySqlConnector;
 
 public sealed class MySqlParameter : DbParameter, IDbDataParameter, ICloneable
 {
+	public static readonly MySqlParameter NullParameter = new MySqlParameter("", null);
```

Comment:
Avoid creating a `public static` property with a value that's a mutable object. User code could run `MySqlParameter.NullParameter.Value = 42`.

Reply:
correct, i'll correct that